src/main/resources/META-INF/resources/webjars/service/prov/prov.js

```diff
@@ -3884,19 +3886,27 @@ define(function () {
 							url: REST_PATH + 'service/prov/' + current.model.subscription + '/storage-lookup?' + type + '=' + qi.id,
 							dataType: 'json',
 							data: function (term) {
+								const regex=/(([\d]+)\s*[*x]\s*)?(\d+)/
+								term =term.match(regex)
```

Comment:
Assign the result to a new variable since the original type of 'term' is String and the new type is a RexExp